app-misc/dateutils/dateutils-0.4.7.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="command line date and time utilities"
+HOMEPAGE="https://www.fresse.org/dateutils/ https://github.com/hroptatyr/dateutils"
+
+case "${PV}" in
+	9999)
+		inherit autotools git-r3
+		EGIT_REPO_URI="https://github.com/hroptatyr/dateutils.git"
+		;;
+	*)
+		SRC_URI="https://github.com/hroptatyr/dateutils/releases/download/v${PV}/${P}.tar.xz"
+		KEYWORDS="~amd64 ~x86"
+esac
+
+LICENSE="BSD"
+SLOT="0"
+
+DEPEND="app-arch/xz-utils
```

Comment:
I'm so sorry to be a pain. I should've noticed this earlier. Is this in DEPEND just to unpack the tarball? If so, pop it in BDEPEND. This helps out people doing cross-compilation.

Reply:
You're correct, it isn't needed at runtime. I'll move it to BDEPEND.